### PR TITLE
feat: update search box style and config

### PR DESCRIPTION
**Pull request by mnavarroespinosa**
EX-6341

### CODE REVIEW

File: src/components/search-box.vue [hunk 1]
```diff
@@ -1,23 +1,36 @@
 <template>
-  <div
-    class="x-input-group"
-    :class="$x.device === 'mobile' ? 'x-input-group--card' : 'x-input-group--line'"
-  >
-    <SearchIcon />
-    <SearchInput :placeholder="$t('searchBox.placeholder')" />
-    <ClearSearchInput>{{ $t('searchBox.clear') }}</ClearSearchInput>
+  <div class="x-input-group x-input-group--card">
```
Comment (diegopf) [inline src/components/search-box.vue:2]: The border color should be the lead one, I think. 
```suggestion
  <div class="x-input-group x-input-group--card x-border-color--lead">
```

File: src/components/search-box.vue [hunk 2]
```diff
@@ -1,23 +1,36 @@
...
+    <SearchInput
+      :placeholder="$t('searchBox.placeholder')"
```
Comment (diegopf) [inline src/components/search-box.vue:4]: I think we can remove the placeholder, at least according to figma designs

File: src/components/search-box.vue [hunk 3]
```diff
@@ -1,23 +1,36 @@
...
+      :class="$x.device === 'mobile' ? 'x-padding--left-05' : 'x-padding--left-08'"
+    />
+
+    <ClearSearchInput v-if="$x.device === 'desktop' && $x.query.searchBox" class="x-font-size--04">
```
Comment (diegopf) [inline src/components/search-box.vue:8]: Currently the default font weight is 300 and I think it should be 400. I think you can use the `--x-number-font-weight-input-group-default-button` token to adjust it

**Review by diegopf — CHANGES_REQUESTED** [review]

File: src/components/search-box.vue (hunk 1 above)
Comment (tiborux) [reply src/components/search-box.vue:2]: yes, you are right

File: src/components/search-box.vue [hunk 4]
```diff
@@ -1,23 +1,36 @@
...
-    class="x-input-group"
-    :class="$x.device === 'mobile' ? 'x-input-group--card' : 'x-input-group--line'"
-  >
-    <SearchIcon />
-    <SearchInput :placeholder="$t('searchBox.placeholder')" />
-    <ClearSearchInput>{{ $t('searchBox.clear') }}</ClearSearchInput>
+  <div class="x-input-group x-input-group--card">
+    <SearchInput
+      :placeholder="$t('searchBox.placeholder')"
+      :class="$x.device === 'mobile' ? 'x-padding--left-05' : 'x-padding--left-08'"
```
Comment (tiborux) [inline src/components/search-box.vue:5]:
```suggestion
      :class="$x.device === 'mobile' ? 'x-padding--left-06' : 'x-padding--left-08'"
```

Should be 24px

**Review by tiborux — CHANGES_REQUESTED** [review]

File: src/components/search-box.vue (hunk 4 above)
Comment (tiborux) [reply src/components/search-box.vue:5]: it was a mistake in the designs

File: src/components/search-box.vue (hunk 3 above)
Comment (mnavarroespinosa) [reply src/components/search-box.vue:8]: Sure, I've added it!

**Review by tiborux — APPROVED** [review]

**Review by diegopf — APPROVED** [review]

**Pull request merged**